lib/chess/core/pieces/pawn.rb

```diff
@@ -12,12 +12,13 @@ def self.create_and_occupy(team, current_cell)
         end
 
         def can_move_to?(target_cell, cells)
-          deltas = attacking?(target_cell) ? attack_deltas : move_deltas
-
           return false if target_cell.team == @team
-          return evaluate_with_one_move(target_cell, cells, deltas) if can_move_only_once_at_time?
 
-          evaluate_with_path(target_cell, cells, deltas)
```

Comment:
It doesn't need this #can_move_only_once_at_time? check, because this is a Pawn's #can_move_to? overwritten